README.md

```diff
@@ -100,6 +100,26 @@ $ docker volume rm MyVolume
 $ /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py volume ls
 ```
 
+## Upgrade Instructions
+
+User's data is not impacted during driver or plugin upgrade.
+vDVS managed plugin is taking care of the control path (management layer) and doesn't deal with data path. So during the upgrade process, only management related operations will be affected and data will be intact. Management operations like attach/detach/create/delete volume using vDVS driver may run into an error for the time being until the upgrade process is finished.
+
+Steps for safe upgrade:
+1. Uninstall VIB (vDVS driver)
+```esxcli software vib remove -n esx-vmdkops-service```
+2. Download and install the latest VIB from the vDVS release page
+```esxcli software vib install -v /tmp/<vib_name>.vib```
+3. Uninstall the managed plugin from the docker host
+```docker plugin rm -f <plugin name>```
+4. Install the latest release of vDVS managed plugin
+```docker plugin install --grant-all-permissions --alias vsphere vmware/docker-volume-vsphere:latest```
+
+We do not require user to stop containers to do plugin upgrade.
+If plugin upgrade was done while volume was still mounted to a container, user may not see the attached volume after upgrade. Docker restart will fix the issue and all volumes will start getting listed.[Issue #1753](https://github.com/vmware/docker-volume-vsphere/issues/1753)
```

Comment:
@govint  I verified that if we change the alias name during plugin install and use a newer alias name, I didn't have to restart docker to use the volumes.

Please see the steps below:
```
root@sc-rdops-vm02-dhcp-52-237:~# docker plugin install --grant-all-permissions --alias vspherexxxxx vmware/docker-volume-vsphere:0.15
0.15: Pulling from vmware/docker-volume-vsphere
967f49dfadaf: Download complete
Digest: sha256:64a50830c2fa9befbede6efca3f28d2e2d8b4e595703f1b5984470579656e935
Status: Downloaded newer image for vmware/docker-volume-vsphere:0.15
Installed plugin vmware/docker-volume-vsphere:0.15
root@sc-rdops-vm02-dhcp-52-237:~# docker plugin ls
ID                  NAME                  DESCRIPTION                           ENABLED
5b4d7b7f3c38        vspherexxxxx:latest   VMWare vSphere Docker Volume plugin   true
root@sc-rdops-vm02-dhcp-52-237:~#
root@sc-rdops-vm02-dhcp-52-237:~# docker volume create -d vspherexxxxx --name TestVolumexxxxx
TestVolumexxxxx
root@sc-rdops-vm02-dhcp-52-237:~# docker volume ls
DRIVER                VOLUME NAME
vspherexxxxx:latest   TestAAA@shared    Vmfs   -   0
vspherexxxxx:latest   TestVolume123@shared    Vmfs   -   0
vspherexxxxx:latest   TestVolumexxxxx@shared    Vmfs   -   0
vspherexxxxx:latest   vsphere15VOL1@shared    Vmfs   -   0
root@sc-rdops-vm02-dhcp-52-237:~# docker plugin ls
ID                  NAME                  DESCRIPTION                           ENABLED
5b4d7b7f3c38        vspherexxxxx:latest   VMWare vSphere Docker Volume plugin   true
root@sc-rdops-vm02-dhcp-52-237:~#
root@sc-rdops-vm02-dhcp-52-237:~# docker plugin rm -f vspherexxxxx
vspherexxxxx
root@sc-rdops-vm02-dhcp-52-237:~# docker plugin install --grant-all-permissions --alias vsphereyyyyy vmware/docker-volume-vsphere:0.16
0.16: Pulling from vmware/docker-volume-vsphere
1d54a760dc2a: Download complete
Digest: sha256:5281d47506740ee927001ea5fc16d8584ed7fc04de0fb25969f0b11eb3243345
Status: Downloaded newer image for vmware/docker-volume-vsphere:0.16
Installed plugin vmware/docker-volume-vsphere:0.16
root@sc-rdops-vm02-dhcp-52-237:~#
root@sc-rdops-vm02-dhcp-52-237:~# docker plugin ls
ID                  NAME                  DESCRIPTION                           ENABLED
22380aa655b0        vsphereyyyyy:latest   VMWare vSphere Docker Volume plugin   true
root@sc-rdops-vm02-dhcp-52-237:~#
root@sc-rdops-vm02-dhcp-52-237:~#
root@sc-rdops-vm02-dhcp-52-237:~#
root@sc-rdops-vm02-dhcp-52-237:~# docker volume ls
DRIVER                VOLUME NAME
vsphereyyyyy:latest   TestAAA@shared    Vmfs   -   0
vsphereyyyyy:latest   TestVolume123@shared    Vmfs   -   0
vsphereyyyyy:latest   TestVolumexxxxx@shared    Vmfs   -   0
vsphereyyyyy:latest   vsphere15VOL1@shared    Vmfs   -   0
root@sc-rdops-vm02-dhcp-52-237:~#
root@sc-rdops-vm02-dhcp-52-237:~# docker run -it --volume-driver=vsphereyyyyy:latest -v 'TestVolumexxxxx@shared    Vmfs   -   0':/vol1 --name ub_xxxxx ubuntu
root@ff3b0ec54308:/#
```